lib/galaxy/tool_util/parser/parameter_validators.py

```diff
@@ -668,9 +668,9 @@ def raise_error_if_valiation_fails(
         raise AssertionError("Validator logic problem - computed validation value must be boolean")
     if message is None:
         message = validator.message
-    if message is None:
+    if not message:
         message = DEFAULT_VALIDATOR_MESSAGE
-    assert message
+    assert message is not None
```

Comment:
I don't think there's a possibility message can be None or fals-y now ?
```suggestion
```

Reply:
spent too much time in typescript, of course mypy still sees it as Optional[str]

Reply:
Someday 😅